### ISSUE TITLE
Provide test data

### ISSUE THREAD

**Issue by richelbilderbeek** [icbi-lab/nextNEOpi#33]
Dear nextNEOpi maintainers,

Thanks for nextNEOpi, it seems like a useful tool!

However, I would have enjoy to have seen some more simulated/public example data, so anyone can verify that everything works as it should be. I see that there are some example files, but running it fails.

For example, when running ...

```
nextflow run nextNEOpi.nf --batchFile example_batchFile_FASTQ.csv \
  -profile singularity \
  -config conf/params.config \
  --exomeCaptureKit "Twist" \
  --trim_adapters true \
  --trim_adapters_RNAseq true \
  --mutect2ponFile 1000g_pon.hg38.vcf.gz \
  --use_sequenza_cnvs true \
  --TCR false \
  --accept_license
```

... then these files are needed as well:

 * `reads_s1_t_1.fastq.gz`
 * `reads_s1_t_2.fastq.gz`
 * `reads_s1_n_1.fastq.gz`
 * `reads_s1_n_2.fastq.gz`
 * `reads_s1_r_1.fastq.gz`
 * `reads_s1_r_2.fastq.gz`
 * `data/sample2_hla.txt`

It would be great to have simulated/public data added, among others to add Continuous Integration and/or help users report bugs.

For me, [this script](https://github.com/richelbilderbeek/ticket_267886/blob/master/scripts/run.sh) really looks a bit too clumsy :-/

Thanks and cheers, Richel  Bilderbeek

**Comment by richelbilderbeek** [comment]
See also https://github.com/hammerlab/wobidisco/issues/8

**Comment by richelbilderbeek** [comment]
See also https://github.com/jiujiezz/tsnad/issues/32

**Comment by riederd** [comment]
Hi,

thanks for your interest in `nextNEOpi` and for your suggestion to include test data. We will consider it for future releases. For now we only test with our in house patient data which - due to privacy reasons - we can not share.

**Comment by richelbilderbeek** [comment]
Note to self: testing datasets can be found at https://github.com/nf-core/test-datasets

**Comment by riederd** [comment]
closed by 92712e0748a9cc1b0ec5755e2d0c9f5d46a224fa
